packages/hub/src/app/models/[owner]/[slug]/FallbackLayout.tsx

```diff
@@ -20,8 +20,10 @@ export const FallbackModelLayout: FC<Props> = ({ username, slug }) => {
         // This causes an entity node to not have an icon or a link until the owner type is known.
         { slug: username },
         slug,
-        variableName,
-        "EXPORT"
+        {
+          name: variableName,
+          type: "EXPORT",
```

Comment:
Can you check for `isModelRelativeValuesRoute(pathname)` here too, like in `ModelLayout`? (I mentioned this in previous review, maybe you missed it)

Or, even better, extract it to a common custom hook:
```typescript
function useEntityNodesVariable() {
  const { variableName } = useParams<{ variableName: string }>();
  const pathname = usePathname();
  return {
    name: variableName,
    isModelRelativeValuesRoute(pathname) ? "RELATIVE_VALUE": "EXPORT",
  };
}
```

(tbh I'd go one step further and fold `entityNodes` into `EntityLayout` component or convert it to a hook, because the logic for building this data is too tightly coupled with URL, for which you need to be in hook context, so it's a problem that `entityNodes` is a plain function that doesn't have access to `useParams` and `usePathname`)

Reply:
Resubmitted